features/step_definitions/order_steps.rb

```diff
@@ -14,3 +14,29 @@
 And(/^I fill in Delivery Date with "([^"]*)"$/) do |date|
   fill_in('order_delivery_date', with: date)
 end
+
+
+Given(/^"([^"]*)"'s order contains:$/) do |billing_name, table|
+  @order = Order.find_by(billing_name: billing_name)
```

Comment:
isn't better to create an `billing_id`and use that instead of using `billing_name`?

Reply:
In test? I don't see a need for that